about.html

```diff
@@ -7,67 +7,95 @@
-    
-    Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.
-
-    Learn More
-
-    Let's Work Together
-
-    Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.
+          <h2>
+            About
+          </h2>
+
+            <p>Leverage agile frameworks to provide a robust synopsis for high level overviews. Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition. Organically grow the holistic world view of disruptive innovation via workplace diversity and empowerment.</p>
+
+      </section>
+
+      <section class="middle">
+        <div class="boxing">
+          <div class="finger-img"><img class="point"src="https://github.com/Lfritze/Sprint-Challenge--User-Interface/blob/master/img/about-plan.png?raw=true" alt="strategy"></div>
+            <h2>
+              Strategy
+            </h2>
+              <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+              <a href="#">Learn More</a>
+        </div> <!-- closes boxing div -->
+      
+          <div class="boxing">
+            <div class="coder-img"><img class="worker"src="https://github.com/Lfritze/Sprint-Challenge--User-Interface/blob/master/img/about-working.png?raw=true" alt="strategy"></div>
+              <h2>
+                How We Work
+              </h2>
+                <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                  <a href="#">Learn More</a>
+      </div><!-- closes boxing div -->
+        
+      <div class="boxing">  
+        <div class="office-img"><img class="office"src="https://github.com/Lfritze/Sprint-Challenge--User-Interface/blob/master/img/about-office.png?raw=true" alt="strategy">        </div>
+          <h2>Places We Work</h2>
+              <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+                <a href="#">Learn More</a>
+      </div><!-- closes boxing div -->
+   
+      <div class="boxing">
+        <div class="collaborate-img"><img class="people-pic"src="https://github.com/Lfritze/Sprint-Challenge--User-Interface/blob/master/img/about-meeting.png?raw=true" alt="strategy">
+        </div>
+          <h2>Collaboration</h2>    
+            <p>Iterative approaches to corporate strategy foster collaborative thinking to further the overall value proposition.</p>
+              <a href="#">Learn More</a>
+      </div><!-- closes boxing div -->
+    </section> <!-- closes middle section -->
+      
+    <section class="bottom-part">
```

Comment:
what properties would you need to add to make the content appear to be 'pinched in' from the edges of the container?